cognite/replicator/replication.py

```diff
@@ -194,6 +195,9 @@ def make_objects_batch(
                 update_objects.append(dst_obj)
             else:
                 unchanged_objects.append(dst_obj)
+        elif dst_ts:
+            if any(x for x in dst_ts if x.external_id == src_obj.external_id):
```

Comment:
We have a loop within a loop here, which could potentially create run-time complexity problems. Maybe it is better to make a set of external_ids in dst_ts outside the loop, as that would allow you a 2(n) instead of (o)^2 runtime.

Reply:
Indeed, great suggestion, thanks for the feedback!

Reply:
Should be fixed, creating a set before the for loop and fills the set with the external ids if the parameter exists